sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/MemoryStateStore.scala

```diff
@@ -30,7 +30,7 @@ class MemoryStateStore extends StateStore() {
   }
 
   override def createColFamilyIfAbsent(colFamilyName: String): Unit = {
-    throw new UnsupportedOperationException("Creating multiple column families is not supported")
+    throw StateStoreErrors.multipleColumnFamilies("MemoryStateStoreProvider")
```

Comment:
Is it possible to augment more tests to ensure that the NERF is being used correctly ?